charts/tekton-apps/Chart.yaml

```diff
@@ -13,7 +13,7 @@ triggersVersions: "v0.16.0"
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.26
+version: 0.3.0-dev
```

Comment:
Why there is such a big jump in versions? From 0.1.26 -> 0.3.0?

Reply:
agreed. should next incremental version @populov.
we will introduce 0.2.0 after I finish upgrading the underlying tekton-engine charts.

Pls use `0.1.28` and remove tag 0.3.0-dev from the remote and from rocks/values.yaml